ccp/src/main/res/raw/a_new_lang_template.xml

```diff
@@ -26,12 +26,12 @@
             name="TODO: replace with  translated country name"
             english_name="Antigua and Barbuda"
             name_code="ag"
-            phone_code="1" />
+            phone_code="1-268" />
```

Comment:
I picked up this request from someone in the issues list. 
I used the reference from https://countrycode.org/, https://en.wikipedia.org/wiki/North_American_Numbering_Plan also here https://en.wikipedia.org/wiki/List_of_country_calling_codes
Most of the american territories have only one area code so I thought it could be added, maybe without the dash "-"?
I'll remove it for now and commit.